`go vet` is failing locally

```
$ make test
go get github.com/mattn/goveralls
go get github.com/modocache/gover
go get github.com/Masterminds/glide
go get github.com/sgotti/glide-vc
go get github.com/golang/lint/golint
go get github.com/mitchellh/gox
./script/check-vendor.sh
Checking for nested vendor dirs
OK
Checking if vendor was cleaned using glide-vc.
OK
./script/check-gofmt.sh
gofmt OK
go vet ./pkg/... ./cmd/... .
pkg/loader/compose/compose_test.go:59: arg expected for printf verb %s of wrong type: github.com/kubernetes/kompose/pkg/kobject.HealthCheck
exit status 1
Makefile:75: recipe for target 'vet' failed
make: *** [vet] Error 1
```

--- a/pkg/loader/compose/compose_test.go
+++ b/pkg/loader/compose/compose_test.go
@@ -56,7 +56,7 @@ func TestParseHealthCheck(t *testing.T) {
 	}
 
 	if !reflect.DeepEqual(output, expected) {
-		t.Errorf("Structs are not equal, expected: %s, output: %s", expected, output)
+		t.Errorf("Structs are not equal, expected: %v, output: %v", expected, output)
 	}
 }
 

